src/modules/exampleEntity/domain/exampleEntity.entity.ts

```diff
@@ -0,0 +1,50 @@
+import { createId } from "@paralleldrive/cuid2"
+interface ExampleEntityProps {
+    name: string;
+    email: string;
+    birthDate: Date;
+    id?: string;
+}
+export class ExampleEntity{
+    id: string
```

Comment:
Run lint para padronizar o estilo
```suggestion
    id : string
```